docs/dashboard_config.md

```diff
@@ -39,6 +40,7 @@ spec:
     disableUserManagement: false
     disableProjects: false
     disableModelServing: false
+    disableCustomServingRuntimes: false
```

Comment:
mmmm right, but I think that might be out of scope this PR, there's a bigger issue with this since the CRD object of the odhDashboard has de default to true, I'm gonna resolve this conversation and log a new issue.

Reply:
https://github.com/opendatahub-io/odh-dashboard/issues/1106